src/commands/support/reportedissues.ts

```diff
@@ -1,17 +1,83 @@
+import { Colors, EmbedField } from 'discord.js';
 import { CommandDefinition } from '../../lib/command';
 import { CommandCategory } from '../../constants';
 import { makeEmbed } from '../../lib/embed';
+import Logger from '../../lib/logger';
+
+const jsdom = require('jsdom');
+
+const { JSDOM } = jsdom;
```

Comment:
```suggestion
import { Colors, EmbedField } from 'discord.js';
import { JSDOM } from 'jsdom';
import { CommandDefinition } from '../../lib/command';
import { CommandCategory } from '../../constants';
import { makeEmbed } from '../../lib/embed';
import Logger from '../../lib/logger';
```
TypeScript import using TS types and also import sorting for lint